docs/_articles/en/lwc/linting.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Linting 
+lang: en
+---
+
+Linting provides errors about malformed code while you edit. VS Code enforces Salesforce’s ESLint rules. 
+
+## Prerequisites
+
+[The active LTS release](https://nodejs.org/en/about/releases/) of Node.js built with SSL support. 
+
+## Installation
+
+**For a New Project**
+
+If you've recently created a new SFDX project using the `force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included. Make sure to run `npm install` in your project directory to install the dependencies from `package.json`. To start linting, run `npm run lint:lwc`.
+
+**For an Existing Project**
+
+1. Check whether your project contains a `package.json` file. This file specifies the packages and version required to run your project.
+ 
+If you already have a `package.json` file:
+ 
+- Add `lint` and `lint:lwc` to the scripts section of `package.json`.
+
+```
+"scripts": {
+	"lint": "npm run lint:lwc",
+  	"lint:lwc": "eslint force-app/main/default/lwc"
+}
+```
+
+- Add the Salesforce ESLint configuration and ESLint to devDependencies. 
+	
+```
+"devDependencies" {
+	"@salesforce/eslint-config-lwc": "0.4.0",
+	"eslint": "^5.16.0"
+}
+```
+ 
+- Run `npm install` on your project directory to install the dependencies. Start linting by running `npm run lint:lwc`.
+
+   If you don't have a `package.json` file, copy this file and save it in your project.
+
```

Comment:
```suggestion
   Copy this file and save it in your project.
```

Reply:
Which file?

Reply:
I updated this in my most recent commit

Reply:
can we explicit and name the link, e.g., "Copy this package.json file"